cloudapi/v1/cloudapi.proto

```diff
@@ -17,11 +20,9 @@ message Instance {
   string id = 1;
   string project_name = 2;
   string type = 3;
```

Comment:
also duplicate. we have InstanceDetail.

either flatten InstanceDetail into the top level, or remove this